.gitignore

```diff
@@ -4,3 +4,6 @@
 
 # External files
 /vendor
+
+# IDE's
+/.vscode
```

Comment:
Editor preferences belongs in the user's own git configuration, not the repository's.

git has a per-repository user config file at `.git/info/exclude` and user-wide at `~/.config/git/ignore`.